addon/mixins/debug-logging.js

```diff
@@ -0,0 +1,68 @@
+import Ember from 'ember';
+const { getWithDefault, get } = Ember;
+import Debug from '../utils/debug';
+
+// Keep this object around to keep track of logs.
+const DebugLogging = Ember.Object.create({
+  loggers: new Ember.Map(),
```

Comment:
why a map? The docs say the only diff from a POJO is that the keys don't have to be strings.